article - migrate helm charts

gmzabos opened the pull request.

gmzabos committed:
Add article

cmeissner commented on _posts/2021-09-17-migrate-helm-charts.md:
```diff
@@ -0,0 +1,80 @@
+tags: [kubernetes, openshift, helm, chart, repository, redhat, quay, registry]
+author: gmzabos
+---
+
```
I missed a headline for that paragraph. May be you can add a descriptive headline here. 

It can be ` What is the problem?` to stay in the style of asking questions in headlines like below

cmeissner replied:
I modified the text a bit. So I think we don't need a heading here.

cmeissner committed:
Review and rewrite article

Rewrite article opener to make it more identifying for the reader.
Shorten the headings of the 5 steps to a minimum and add descriptive
text to each step.
Fix some heading issues, we never use h1 (#) in our posts as the only
h1 is the title itself.
